README.md

```diff
@@ -1,26 +1,45 @@
-# Coconut Online Interpreter [![Inline docs](http://inch-ci.org/github/cs121-team-panda/coconut-interpreter-flask.svg?branch=master)](http://inch-ci.org/github/cs121-team-panda/coconut-interpreter-flask) [![CircleCI](https://circleci.com/gh/cs121-team-panda/coconut-interpreter-flask/tree/master.svg?style=svg)](https://circleci.com/gh/cs121-team-panda/coconut-interpreter-flask/tree/master)
+# [Coconut Online Interpreter][interpreter-url] [![CircleCI][circleci-image]][circleci-url]
 
-[Coconut](http://coconut-lang.org/) is a functional programming language that compiles to Python. This project is an online interpreter for Coconut. On the webpage, the user can interact with the code editor and click the run button to execute their program and view the output.
+[circleci-image]: https://circleci.com/gh/cs121-team-panda/coconut-interpreter.svg?style=shield
+[circleci-url]: https://circleci.com/gh/cs121-team-panda/coconut-interpreter
+[interpreter-url]: https://cs121-team-panda.github.io/coconut-interpreter
 
-Try the current version at: [https://cs121-team-panda.github.io/coconut-interpreter/](https://cs121-team-panda.github.io/coconut-interpreter/)
-
-Developed by Jonathan Cruz, Teerapat Jenrungrot, Natalie Kadonaga, and Brittany Wang.
+[Coconut](http://coconut-lang.org/) is a functional programming language that compiles to Python. The online interpreter brings an instant programming environment for Coconut. Users can enter and execute Coconut code and view the output.
```

Comment:
"The online interpreter brings... for Coconut" is a little awkward, since "bring" isn't usually followed by the preposition "for"

maybe, "The online interpreter provides"?